tests/trainer/test_trainer.py

```diff
@@ -3990,91 +3988,12 @@ def test_torchdynamo_full_eval(self):
             del trainer
             torchdynamo.reset()
 
-            # 3. TorchDynamo nvfuser
-            trainer = get_regression_trainer(a=a, b=b, eval_len=eval_len, torchdynamo="nvfuser", output_dir=tmp_dir)
-            metrics = trainer.evaluate()
-            self.assertAlmostEqual(metrics["eval_loss"], original_eval_loss)
-            torchdynamo.reset()
-
-            # 4. TorchDynamo fx2trt
+            # 3. TorchDynamo fx2trt
             trainer = get_regression_trainer(a=a, b=b, eval_len=eval_len, torchdynamo="fx2trt", output_dir=tmp_dir)
             metrics = trainer.evaluate()
             self.assertAlmostEqual(metrics["eval_loss"], original_eval_loss)
             torchdynamo.reset()
 
-    @unittest.skip(reason="torch 2.0.0 gives `ModuleNotFoundError: No module named 'torchdynamo'`.")
-    @require_torch_non_multi_gpu
-    @require_torchdynamo
-    def test_torchdynamo_memory(self):
```

Comment:
This test compares nvfuser to eager, nvfuser was removed: https://github.com/pytorch/pytorch/pull/105789

(the test was also being skipped 👀 )